project/build.scala

```diff
@@ -11,6 +11,15 @@ import sbtrelease.ReleasePlugin._
 import scalariform.formatter.preferences._
 
 object ChaosBuild extends Build {
+  lazy val chaosExamples: Project = Project(
+    id = "chaos-examples",
+    base = file("chaos-examples"),
+    settings = baseSettings ++
+      formatSettings ++
+      publishSettings ++
+      revolverSettings
```

Comment:
@aquamatthias I did add revolverSettings right here. Did you mean some other place and I am misunderstanding it?